src/DropdownLayout/test/DropdownLayout.visual.js

```diff
@@ -157,3 +182,24 @@ tests.forEach(({ describe, its }) => {
     );
   });
 });
+
+rtl.forEach(({ describe, its }) => {
+  its.forEach(({ it, props }) => {
+    storiesOf(`DropdownLayout ${describe ? '/' + describe : ''}`, module).add(
+      it,
+      () => (
+        <div
+          style={
+            props.inContainer
+              ? containerStyles
+              : { width: '240px', margin: 'auto' }
+          }
+        >
+          <RTLWrapper rtl={rtl}>
```

Comment:
Maybe we can just run all tests for regular DropdownLayout and RTL DropdownLayout.
So in one screenshot we can have the regular version and the rtl one.

Also, rtl is a boolean so you can just use it this way:
```suggestion
          <RTLWrapper rtl>
```